src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+    private static final Random random = new Random();
+    private static final ColorSupplier color = new ColorSupplier();

```

Comment:
Please, choose a more appropriate naming, because this variable doesn't hold **color**, it holds **colorSupplier**